components/Snackbar/src/MDCSnackbarMessageView.m

```diff
@@ -924,6 +924,22 @@ - (NSArray *)horizontalButtonLayoutConstraints {
   return constraints;
 }
 
+- (void)activateConstraintsForAlignment:(MDCSnackbarAlignment)alignment {
+  switch (alignment) {
+    case MDCSnackbarAlignmentCenter:
+      self.leadingConstraint.active = NO;
+      self.centerConstraint.active = YES;
+      break;
+    case MDCSnackbarAlignmentLeading:
+      self.leadingConstraint.active = YES;
+      self.centerConstraint.active = NO;
+      break;
+    default:
+      NSAssert(NO, @"`alignment` should be a valid MDCSnackbarAlignment value");
+      break;
```

Comment:
Please add default behavior here for non-debug releases. I would propose the `MDCSnackbarAlignmentCenter` behavior should be used.